sklego/pipeline.py

```diff
@@ -301,3 +303,78 @@ def log_callback(self, func):
         self._log_callback = func
         if self._log_callback == "default":
             self._log_callback = default_log_callback
+
+def _name_estimators(estimators):
```

Comment:
Why not use the `_name_estimators` from sklearn? Am I missing something?

Reply:
Good catch, not missing anything. Importing it is simply better. I've made the suggested change and made a commit for it. Thanks Cor :-)